Refactor snipe.js and speedtest.js

RfadnjdExt opened the pull request:
This pull request refactors the `snipe.js` and `speedtest.js` files. 

In `snipe.js`, the `actionRow` variable declaration has been moved outside of the `createEmbedAndButtons` function to improve scope management.

In `speedtest.js`, an underscore has been added before the `client` parameter in the `run` function to indicate that it's not being used.

These changes enhance the code structure and maintainability. Please review and merge this pull request at your convenience.

RfadnjdExt committed:
Refactor snipe.js and speedtest.js

- In snipe.js:
  - Moved the declaration of `actionRow` variable outside of the `createEmbedAndButtons` function for better scope management.
- In speedtest.js:
  - Added an underscore before the `client` parameter in the `run` function to indicate it's not being used.

RfadnjdExt committed:
Merge branch 'main' of https://github.com/RfadnjdExt/stardust

Aruh1 commented:
```if (currentPage !== filteredSnipeData.length) {
                    buttons.push(
                        new ButtonBuilder().setCustomId(nextButtonCustomID).setEmoji("▶").setStyle(ButtonStyle.Primary)
                    );
                }```

```
/workspaces/stardust/commands/snipe.js:56
                    buttons.push(
                    ^

ReferenceError: buttons is not defined
    at createEmbedAndButtons (/workspaces/stardust/commands/snipe.js:56:21)
    at Object.run (/workspaces/stardust/commands/snipe.js:67:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.16.1

RfadnjdExt commented:
> ```
>                     buttons.push(
>                         new ButtonBuilder().setCustomId(nextButtonCustomID).setEmoji("▶").setStyle(ButtonStyle.Primary)
>                     );
>                 }```
> ```
> 
> /workspaces/stardust/commands/snipe.js:56 buttons.push( ^
> 
> ReferenceError: buttons is not defined at createEmbedAndButtons (/workspaces/stardust/commands/snipe.js:56:21) at Object.run (/workspaces/stardust/commands/snipe.js:67:33) at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
> 
> Node.js v18.16.1

wokawok

RfadnjdExt committed:
Refactor config.sample.js

This commit removes the config.sample.js file as it is no longer needed. The file contained configuration settings for the bot, including token, owner ID, MongoDB URL, and other options. These settings can now be managed directly in the bot's configuration file.

This update simplifies the project structure and improves security by removing sensitive information from the sample file.
